palette/benches/cie.rs

```diff
@@ -0,0 +1,92 @@
+use criterion::{criterion_group, criterion_main, Criterion, Throughput};
+use palette::convert::FromColorUnclamped;
+use palette::{Lab, Lch, Xyz, Yxy};
+
+#[path = "../tests/convert/data_color_mine.rs"]
+#[allow(dead_code)]
+mod data_color_mine;
+use data_color_mine::{load_data, ColorMine};
+
+/* Benches the following conversions:
+    - xyz to lab
+    - lch to lab
+    - lab to lch
+    - rgb to xyz
+    - yxy to xyz
+    - lab to xyz
+    - xyz to yxy
+*/
+
+fn cie_conversion(c: &mut Criterion) {
+    let mut group = c.benchmark_group("Cie family");
+    let colormine: Vec<ColorMine> = load_data();
+    let lab: Vec<Lab> = colormine
+        .iter()
+        .map(|x| Lab::from_color_unclamped(x.xyz))
+        .collect();
+    let lch: Vec<Lch> = colormine
+        .iter()
+        .map(|x| Lch::from_color_unclamped(x.xyz))
+        .collect();
+
+    group.throughput(Throughput::Elements(colormine.len() as u64));
+
+    group.bench_with_input("xyz to lab", &colormine, |b, colormine| {
+        b.iter(|| {
+            colormine
+                .iter()
+                .map(|c| Lab::from_color_unclamped(c.xyz))
+                .collect::<Vec<_>>()
```

Comment:
I believe the `Vec` creation should be either moved out or changed to use a loop and `black_box`, so we don't measure the time it takes to allocate and write to the `Vec` too. Maybe something like this for this one:

```rust
for color in &colormine {
  black_box(Lab::from_color_unclamped(c.xyz));
}
```